### PR TITLE
v0.8.0

### PR DESCRIPTION
**Pull request by ELundby45**
Update to version compatible with Python 3.11

https://github.com/jborean93/pyspnego/blob/v0.8.0/pyproject.toml#L3

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,13 +21,13 @@ requirements:
     - {{ compiler('c') }}
   host:
     - python
-    - cryptography
-    - cython
+    - cryptography 39.0.1
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - cryptography 38.0.4
```
(as we don't have 39.0.1 built on 3.11 yet)

**Review by cbouss — APPROVED** [review]
LGTM (with cryptography change)

File: recipe/meta.yaml (hunk 1 above)
Comment (ELundby45) [reply recipe/meta.yaml]: 👍 Good catch. Thank you!

**Review by ryanskeith — APPROVED** [review]
LGTM

**Pull request merged**